.github/pull_request_template.md

```diff
@@ -3,5 +3,6 @@
 <!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
 <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
 - [ ] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
+- [ ] Have you checked script on UBI 9.3 container
```

Comment:
This is good - but can it simply be "on a UBI 9 container"?

Reply:
done